grammars/julia.cson

```diff
@@ -101,10 +101,12 @@ repository:
   function_call:
     patterns: [
       {
-        begin: "([\\w.!]+!?)\\w*\\("
+        begin: "([\\w.!]+!?)\\w*({(?:[^{}]|{(?:[^{}]|{[^{}]*})*})*})?\\("
```

Comment:
Agreed, seems odd to have it. 

Would that first capture group be better as `(\\w+!?)` then skip the trailing `\\w*`?